test/helpers/cons.go

```diff
@@ -257,6 +258,7 @@ var badLogMessages = map[string][]string{
 	NACKreceived:      nil,
 	RunInitFailed:     {"signal: terminated", "signal: killed"},
 	sizeMismatch:      nil,
+	emptyBPFInitArg:   nil,
```

Comment:
I think these changes snuck in from another PR?

Reply:
Oops, indeed. This should have gone into #10230. Will remove.